docs/ossec_alerts.md

```diff
@@ -40,7 +41,6 @@ These are the values you must specify in `prod-specific.yml`:
  * Email username to authenticate to the SMTP relay: `sasl_username`
  * Domain name of the email used to send OSSEC alerts: `sasl_domain`
  * Password of the email used to send OSSEC alerts: `sasl_password`
- * The fingerprint of your SMTP relay: `smtp_relay_fingerprint`
```

Comment:
You should document the three supported tiers of TLS connection security here. Maybe re-include this line but add "(Optional)" at the beginning, add a line for the smtp cafile override, and add a paragraph somewhere explaining how to choose between the three options.